app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -297,6 +302,13 @@ fun AppViewContent(
         modifier = Modifier
           .padding(top = 160.dp)
           .align(Alignment.TopStart)
+          .clickable {
+            if (bundleTitle != null && bundleTag != null) {
```

Comment:
This looks weird.
Better us it as a single value like so:
`if (bonusBundle != null)`